packages/grafana-data/src/themes/createVizColors.ts

```diff
@@ -0,0 +1,177 @@
+import { FALLBACK_COLOR } from '../types';
+import { ThemeColors } from './createColors';
+
+/**
+ * @alpha
+ */
+export interface ThemeVizColors {
```

Comment:
```suggestion
export interface VisualizationColors {
```

Reply:
All the theme sub types are prefix with Theme so want to keep that pattern